src/context/offline.rs

```diff
@@ -64,13 +66,15 @@ impl OfflineAudioContext {
             base,
             length,
             renderer,
+            state: Mutex::new(AudioContextState::Suspended),
         }
     }
 
     /// Given the current connections and scheduled changes, starts rendering audio.
     ///
     /// This function will block the current thread and returns the rendered `AudioBuffer`
     /// synchronously. An async version is currently not implemented.
+    #[allow(clippy::missing_panics_doc)]
     pub fn start_rendering_sync(&mut self) -> AudioBuffer {
```

Comment:
Because we only have an implementation of `start_rendering` that blocks the current thread (hence, `_sync`).
The state after this function is finished is actually `Suspended` again. Or Closed? I'm not actually sure if you should be able to call this function multiple times..